### ISSUE TITLE
Quicksy Multiple Devices Support

### ISSUE THREAD

**Issue by adam-stambaugh** [iNPUTmice/Conversations#4264]
A friend is using Quicksy to communicate with me over XMPP. They installed Quicksy on their tablet (they already had it set up on their smartphone), thinking they would be able to use it there in addition to their phone, but it sounds like it didn't handle it very well. They can only use it on one device or the other. I recommended they have Quicksy installed on one device and install Conversations on any additional devices, using the Quicksy credentials to log in.

I didn't see what happened on their end, if there was a warning or not. If there isn't a warning, it might be worthwhile for phone numbers that are already registered for preventing issues like that in the future. It could even link to instructions for downloading Conversations instead and logging into it with the Quicksy credentials.

**Comment by licaon-kter** [comment]
You can see the new password on the new device and use it on the old one, right?

**Comment by adam-stambaugh** [comment]
Yeah, they can use the credentials for the new device to log into Conversations on the old device. They lost access to their old messages, though. It's not a major issue, just something new users, that don't understand how Quicksy works, might run into.

**Comment by licaon-kter** [comment]
It works "like the proprietary silos the others are using", meaning as easy, but as broken. :)
